.buildkite/scripts/run-win-tests.ps1

```diff
@@ -30,7 +29,7 @@ function installGoDependencies {
 }
 
 fixCRLF
-withGolang $env:GO_VERSION_CHOCO
+withGolang $env:GO_VERSION
```

Comment:
Good news, the gvm install works!
```
-- Install golang --
--
  | go version go1.21.9 windows/amd64
```

Bad news, the tests (silently) fail:
```

gotestsum : The term 'gotestsum' is not recognized as the name of a cmdlet, function, script file, or operable
--
  | program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
  | At C:\buildkite-agent\builds\bk-agent-prod-gcp-1714061473631994270\elastic\elastic-agent-system-metrics\.buildkite\scri
  | pts\run-win-tests.ps1:37 char:1
  | + gotestsum --format testname --junitfile junit-win-report.xml -- -v ./ ...
  | + ~~~~~~~~~
  | + CategoryInfo          : ObjectNotFound: (gotestsum:String) [], CommandNotFoundException
  | + FullyQualifiedErrorId : CommandNotFoundException
```
I think the installed dependencies are not detected in whatever powershell's $PATH is